core/connection/manager.go

```diff
@@ -127,13 +128,14 @@ func (manager *connectionManager) startConnection(consumerID, providerID identit
 	}
 	cancel = append(cancel, func() { dialog.Close() })
 
-	vpnSession, err := session.RequestSessionCreate(dialog, proposal.ID)
+	vpnSession := session.Session{Config: openvpn_node.VPNConfig{}}
```

Comment:
`Config: ` for `session.Session` struct is declared as an `interface{}`. 
Can it be something except `openvpn_node.VPNConfig{}`?

Reply:
Yes `interface{}` stands for ANY type.
And each transport will be able to put different structure here.